### DISCUSSION TITLE
config

### DISCUSSION THREAD

**Discussion by BtAbdul** [LazyVim/LazyVim#1952] [Q&A] [answered]
HI.
Today i installed LazyVim which is amazing, i don't know you Call him LazyVim.?

Anyway my question is i pressed 'x' to Open LazyExtra and enable all plugin which is disabled by default when you install LazyVim first time.
After enabling and restart LazyVim i lost  home screen related plugins and other plugins following are disabled no
             ○ alpha-nvim
              ○ dashboard-nvim
              ○ flash.nvim
              ○ mini.starter
              ○ tree-sitter-hypr

![IMG_20231105_192816](https://github.com/LazyVim/LazyVim/assets/46757797/31c4a087-f8f1-4586-9198-159674e59175)

How i can enable it???

And also i want to display this home screen which is now disabled
![211285846-0b7bb3bf-0462-4029-b64c-4ee1d037fc1c](https://github.com/LazyVim/LazyVim/assets/46757797/1e846878-ff6a-4117-8735-782f18e60087)

**Comment by dpetka2001** [comment]
This happened because `dashboard.nvim` (the default starter screen Lazyvim plugin) is mutually excluded with `alpha.nvim` and `mini.starter`. So, to get it back, just disable the other 2 extras. Also, `flash.nvim` is disabled because you enabled `leap.nvim` from the extras for the same reasons. I don't think you should enable all the extras at the same time. Just enable the ones you want.

**Reply by BtAbdul** [reply]
How i can enable that plugins ?
I try to find but i couldn't get there is a example.lua  file which has some plugins but i couldn't see dashboard. lua

![Screenshot_2023-11-05-20-49-19-55](https://github.com/LazyVim/LazyVim/assets/46757797/c966a222-6e00-4165-9c4a-f967bd441dc2)

**Reply by dpetka2001** [reply, marked as answer]
Open `LazyExtras` and disable the extras by pressing `x`, just like you did when you enabled them.
